runtime/queries/rust/textobjects.scm

```diff
@@ -0,0 +1,20 @@
+(function_item
+  body: (_) @function.inside) @function.around
+
+(struct_item
+  body: (_) @class.inside) @class.around
+
+(enum_item
+  body: (_) @class.inside) @class.around
+
+(union_item
+  body: (_) @class.inside) @class.around
+
+(trait_item
+  body: (_) @class.inside) @class.around
+
+(impl_item
+  body: (_) @class.inside) @class.around
+
+(mod_item
+  body: (_) @class.inside) @class.around
```

Comment:
Probably want to make all of these `block`

Reply:
Each of them have their own block definitions for the body field like `ordered_field_declaration_list` for struct, `enum_variant_list` for enums, etc; I think using the wildcard keeps it readable.

Also the inside objects currently select the braces in addition to everything inside, should it exclude them ?